src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/TextRepository.cs

```diff
@@ -58,25 +55,11 @@ public async Task<TextResource> Get(string org, string app, string language)
             string id = GetTextId(org, app, language);
             if (!_memoryCache.TryGetValue(id, out TextResource textResource))
             {
-                try
-                {
-                    Uri uri = UriFactory.CreateDocumentUri(DatabaseId, CollectionId, id);
-                    textResource = await Client
-                        .ReadDocumentAsync<TextResource>(
-                            uri,
-                            new RequestOptions { PartitionKey = new PartitionKey(org) });
+                textResource = await Container.ReadItemAsync<TextResource>(id, new PartitionKey(org));
```

Comment:
Repository code needs to handle CosmosException and statusCode NotFound, right?

Reply:
yup. Included the try/catch